src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1038,7 +1038,8 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 languageVersion: languageVersion,
                 preprocessorSymbols: defines.ToImmutableAndFree(),
                 documentationMode: parseDocumentationComments ? DocumentationMode.Diagnose : DocumentationMode.None,
-                kind: SourceCodeKind.Regular
+                kind: SourceCodeKind.Regular,
+                features: features.ToImmutableDictionary(feature => feature, feature => "true")
```

Comment:
My mistake; you should be looking for a `:` (colon) separating the key and value.